spikeextractors/extraction_tools.py

```diff
@@ -788,10 +793,17 @@ def corrected_args(recording, channel_ids=None, start_frame=None, end_frame=None
             warnings.warn("The recording extractor does not have unscaled traces. Returning scaled traces")
             return_scaled = True
 
-        get_traces_correct_arg = func(recording, channel_ids=channel_ids, start_frame=start_frame, end_frame=end_frame,
-                                      return_scaled=return_scaled, **kwargs)
+        traces = func(recording, channel_ids=channel_ids, start_frame=start_frame, end_frame=end_frame,
+                      return_scaled=return_scaled, **kwargs)
+
+        # scaling
+        if recording.has_unscaled and return_scaled:
```

Comment:
@alejoe91 When/where is the attribute `recording.has_unscaled` set?

Reply:
It's defined at the class level (like the `installed`)